### ISSUE TITLE
PEC for Open Wire commands are incorrect

### ISSUE THREAD

**Issue by GunZi200** [foxBMS/foxbms#10]
In this file: https://github.com/foxBMS/foxbms/blob/master/embedded-software/mcu-common/src/module/ltc/ltc.c

```cpp
static const uint8_t ltc2_BC_cmdADOW_PUP_normal_DCP0[4] = {0x03, 0x68, 0x1C, 0x62}; 
static const uint8_t ltc2_BC_cmdADOW_PDOWN_normal_DCP0[4] = {0x03, 0x28, 0xFB, 0xE8}; 
static const uint8_t ltc2_BC_cmdADOW_PUP_filtered_DCP0[4] = {0x03, 0xE8, 0x1C, 0x62};  
static const uint8_t ltc2_BC_cmdADOW_PDOWN_filtered_DCP0[4] = {0x03, 0xA8, 0xFB, 0xE8}; 
```

I noticed none of these PEC codes are correct. Notice also when the commands are different they use the same PEC code which is not possible. I saw other commands have the same issue.

**Comment by foxBMS** [comment]
Dear @GunZi200,

thanks for sharing, we will investigate this closely.
We come back as soon we can have new information.

Best regards,
The foxBMS Team

**Comment by foxBMS** [comment]
Dear @GunZi200,
Thank you again for reporting this issue.
As far as we can see, the PEC for the first two entries that you mention (open-wire-check; normal mode) seem to be correct. The last two entries (open-wire-check; filtered mode) are, however, not correct.

This issue has gone unnoticed as of now, as the standard implementation uses the normal mode. In order to make sure that similar issues do not occur in the future, specific tests for this case have been set up in-house and the wrong PECs have been corrected in the current release ([1.6.5](https://github.com/foxBMS/foxbms/releases/tag/v1.6.5)).
 
Best regards,
The foxBMS Team
